ui/src/components/NavigationBar/hmcc-logo.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="85" height="33" viewBox="0 0 85 33" fill="#000000" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
please put all images inside of public folder in a directory for consistency's sake

Reply:
all img moved to public